src/main/java/gregtech/client/utils/CTMHooks.java

```diff
@@ -10,13 +10,27 @@
 import net.minecraft.util.EnumFacing;
 import net.minecraftforge.client.ForgeHooksClient;
 
+import team.chisel.ctm.api.model.IModelCTM;
+import team.chisel.ctm.client.model.ModelCTM;
+
+import java.lang.reflect.Field;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
-@SuppressWarnings("unused")
 public class CTMHooks {
 
+    private static Field layers;
+
+    static {
+        try {
+            layers = ModelCTM.class.getDeclaredField("layers");
+            layers.setAccessible(true);
+        } catch (NoSuchFieldException e) {
+            GTLog.logger.error("CTMModHooks no such field");
```

Comment:
Can this static block be safely wrapped in a check for CTM being loaded? If not, I think we should either only debug log or not log at all here. Also this message uses `CTMModHooks` instead of `CTMHooks` like the rest.